File: camandline.java
class cammandline
{
    public static void main(String[] args)
    {
        System.out.println("Demonstrate of camand line");

        int No1 = Integer.parseInt(args[0]);
        int No2 = Integer.parseInt(args[1]);

        int Ans = No1 + No2;

        System.out.println("Number of camand line arguments: " +arg.length);

        System.out.println("Addition is :"+Ans);
    }
}

File: camandline.c
#include<stdio.h>
#include<stdlib.h>

int main(int argc, char *argv[]) 
{
   printf("Demonstration of camand line\n");

   int No1 = atoi(argv[1]);
   int No2 = atoi(argv[2]);

   int Ans = No1 + No2;
  printf("number of cmand line argument : %d\n", argc);
   printf("Addition is : %d\n", Ans);
   
    return 0;
}
